File: py/1101.py
from sys import stdin

ranges = []
for line in stdin:
    line = list(map(int, line.split()))
    
    if min(line) <= 0:
        break
    else:
        ranges.append(line)
    
for i in ranges:
    r = range(min(i), max(i) + 1)
    print(*r, f'Sum={sum(r)}', sep=' ')

File: cpp/1101.cpp
#include <iostream>

using namespace std;

int main()
{
    int a, b, sum;

    while (true) {
        sum = 0;
        cin >> a >> b;

        if (a <= 0 || b <= 0) {
            break;
        }

        if (a > b) {
            a = a + b;
            b = a - b;
            a = a - b;
        }

        for (a; a < b; a++) {
            cout << a << ' ';
            sum += a;
        }

        sum += a;
        cout << a << " Sum=" << sum << '\n';
    }

    return 0;
}